test/floki_test.exs

```diff
@@ -147,35 +149,28 @@ defmodule FlokiTest do
       end
     end
 
+    @tag only_parser: Mochiweb
```

Comment:
First I got confused because it should be the entire name, but then I saw the alias. :smile:

Reply:
Believe me I got confused as well but I just followed what you used for the `case` and it worked. Of course I later saw it was aliased 🤗